### DISCUSSION TITLE
Awesome extension!

### DISCUSSION THREAD

**Discussion by securegh** [Curetix/mailflare-extension#25] [General]
Hello, 

I just wanted to thank you for making this awesome extension, it very useful and and drop in replacement for many online aliases service provider. I like the fact that I can manage my aliases directly from from Cloudflare using your extension instead of third-party provider. 
I searched a lot for a tool with similar features and it's usually a paid lock in third-party feature. It was hard to find your extension, it was almost by luck when i just search for "mail  aliases" on GitHub, I believe you could gain more visibility on: https://alternativeto.com
https://www.producthunt.com/
https://discuss.privacyguides.net/
https://www.privacyguides.org/en/email-aliasing/?h=alias    

I hope your app gain significant adoption and support!
Wishing you best of luck!

**Comment by Curetix** [comment]
Thank you for the kind words! I'm currently working on v2 with many improvements, once that is finished I'll think about doing some "marketing" on those platforms.
